### PR TITLE
add promoteModel sdk functionality

### PR DESCRIPTION
**Pull request by HarryDunham**

### CODE REVIEW

File: ggsdk/sdk.py [hunk 1]
```diff
@@ -38,3 +38,12 @@ def save_dataset(self,
             exists (default=False)
         """
         self._core.save_dataset(data_generator, dataset_id, overwrite)
+
+    def promote_model(self, model_name: str, nlp_task: str, environment: str):
+        """Call promote model api.
+
+        :param model_name: Name of the model to promote within cloud storage
```
Comment (alexiudice) [inline ggsdk/sdk.py:48]: @HarryDunham If this is the right way to document user-facing methods could you change the save_dataset call to use this as well? It's using `@param:` instead.

Comment (HarryDunham) [reply ggsdk/sdk.py:48]: @alexiudice Done! This docstring format is [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText), which seems far more ubiquitous than the previous Epytext format which is what you used. Additionally, all the other python projects use reStructuredText so it makes sense to align everything there. Additionally, if you want this parameter docstring scheme to be autogenerated within Pycharm you can go to File -> Settings -> Python Integrated Tools -> Docstrings section and set `Docstring format: reStructuredText` which'll autopopulate the docstring with the methods arguments.

Comment (alexiudice) [reply ggsdk/sdk.py:48]: Thanks for this!

**Review by alexiudice — APPROVED** [review]
Looks good, thanks.

**Review by bradnussbaum — APPROVED** [review]

**Pull request merged**